server/rooms.ts

```diff
@@ -1764,3 +1805,5 @@ export const Rooms = {
 // initialize
 
 Rooms.global = new GlobalRoomState();
+// ensure battle loading happens after the rest of the server is intialized
+setTimeout(() => Rooms.global.loadBattleState(), 1 * 1000);
```

Comment:
What sort of things here rely on the rest of the server needing to be initialized?